src/tools/sharedInputSchemas.ts

```diff
@@ -0,0 +1,3 @@
+export const projectSlugDescription = `The project slug from listFollowedProjects tool (e.g., "gh/organization/project"). When using this option, branch must also be provided.`;
+export const projectSlugDescriptionNoBranch = `The project slug from listFollowedProjects tool (e.g., "gh/organization/project").`;
+export const branchDescription = `The name of the branch currently checked out in local workspace. This should match local git branch. For example: "feature/my-branch", "bugfix/123", "main", "master" etc.`;
```

Comment:
~ we could return the entire params here
```suggestion
export const projectSlug = z
  .string()
  .describe(
    `The project slug from listFollowedProjects tool (e.g., "gh/organization/project"). When using this option, branch must also be provided.`,
  );
export const projectSlugNoBranch = z
  .string()
  .describe(
    `The project slug from listFollowedProjects tool (e.g., "gh/organization/project").`,
  );
export const branch = z
  .string()
  .describe(
    `The name of the branch currently checked out in local workspace. This should match local git branch. For example: "feature/my-branch", "bugfix/123", "main", "master" etc.`,
  );
```
and then do something like this is the places where we're using them 
```ts
import { branch, projectSlug } from '../sharedInputSchemas.js';

export const getBuildFailureOutputInputSchema = z.object({
  projectSlug: projectSlug.optional(),
  branch: branch.optional(),
....
```